scripts/network.js

```diff
@@ -98,76 +98,30 @@ export class ExplorerNetwork extends Network {
      * @returns {Promise<Object>} the block fetched from explorer
      */
     async getBlock(blockHeight, skipCoinstake = false) {
```

Comment:
At this point skipCoinstake logic can be removed, since it was just a network optimization